Source/Services/RPSLS.Web/wwwroot/css/index.css

```diff
@@ -61,12 +63,6 @@
         margin: 1rem;
     }
 
-    .login-section img {
-        height: 100%;
-        grid-row: 1 / span 2;
-        /* position: relative; */
-    }
-
```

Comment:
This was an error from Multiplayer previous development.

Original version stated:
        width: 75%;
        position: relative;

That does the image grow and have a scroll